pulsar-common/src/main/proto/PulsarMarkers.proto

```diff
@@ -60,6 +60,7 @@ message ReplicatedSubscriptionsSnapshot {
     required string snapshot_id = 1;
     optional MarkersMessageIdData local_message_id = 2;
     repeated ClusterMessageId clusters = 3;
+    optional int64 markers_timestamp = 4;
```

Comment:
This doesn't make sense. Comment added to https://github.com/apache/pulsar/pull/24300#discussion_r2090386514

Reply:
Is the above method simpler and more intuitive?

Reply:
It's changing the wireprotocol just to be able to add an implementation specific field, that doesn't make any sense. Please refactor to use a Java record so that we can proceed with the review. It's not much work to do that. See https://github.com/apache/pulsar/pull/24300#discussion_r2090386514

Reply:
In mixed version clusters after upgrades, there would be problems and addressing that in code will add more complexity since you'd have to check whether the field is set.

Reply:
The versions are compatible, but what you said makes sense.